src/qt/optionsdialog.cpp

```diff
@@ -180,6 +181,10 @@ void OptionsDialog::setMapper()
     mapper->addMapping(ui->spendZeroConfChange, OptionsModel::SpendZeroConfChange);
     mapper->addMapping(ui->coinControlFeatures, OptionsModel::CoinControlFeatures);
 
+    mapper->addMapping(ui->addressTypeLegacy, OptionsModel::AddressTypeLegacy);
```

Comment:
Hmm, I think it makes more sense to have the GUI options mapped 1:1 to the `addresstype` parameter. Perhaps this could be something to look into later?

Reply:
Nvm, it's fine for a setting, especially combined with the per address checkbox in #11991.